include/deal.II/lac/trilinos_tpetra_vector.h

```diff
@@ -433,15 +726,11 @@ namespace LinearAlgebra
 
       /**
        * Attempt to perform an operation between two incompatible vector types.
-       *
-       * @ingroup Exceptions
```

Comment:
What happened here?

Reply:
This should not be deleted!
I restored those lines.